lib/_http_outgoing.js

```diff
@@ -931,7 +931,7 @@ function write_(msg, chunk, encoding, callback, fromEnd) {
   let ret;
   if (msg.chunkedEncoding && chunk.length !== 0) {
     len ??= typeof chunk === 'string' ? Buffer.byteLength(chunk, encoding) : chunk.byteLength;
-    msg._send(NumberPrototypeToString(len, 16), 'latin1', null);
+    msg._send(NumberPrototypeToString(len, 16), 'ascii', null);
```

Comment:
I think we shouldn't change this as this will incur unnecessary overhead as node has to parse the string to ensure it's valid ASCII (and we know it will already be valid because we're using the built-in `Number.prototype.toString()`. `latin1` doesn't have such overhead since it's a single byte encoding.

Reply:
I'm confused. ASCI is also single byte

Reply:
Where/Why is there an overhead?

Reply:
Though I do see now that the change is probably unnecessary.

Reply:
We are a bit inconsistent when we use ascii and when we use latin1.

Reply:
> Where/Why is there an overhead?

With 'ascii' node checks for every byte it's within the range 0-127 inclusive. With 'latin1' there's no such check.

O(n) vs. O(0), basically. :-)